bin/wcpay-live-branches/README.md

```diff
@@ -0,0 +1,18 @@
+## WCPay Live Branches
+
+**wcpay-live-branches.user.js** is a Tampermonkey script that allows you to preview a site with a given Pull Request changeset.
+
+### Installation
+
+* Install Tampermonkey or equivalent user script extension.
+* Click https://github.com/Automattic/woocommerce-payments/raw/develop/bin/wcpay-live-branches/wcpay-live-branches.user.js
```

Comment:
Use this link until this PR is merged. 
https://github.com/Automattic/woocommerce-payments/raw/add/jn-userscript-for-live-branch/bin/wcpay-live-branches/wcpay-live-branches.user.js